docs/api/jax.rst

```diff
@@ -6,6 +6,17 @@
 Jax
 =======
 
+Pre-defined Variable of Logical Axes
+------------------------------------
+* BATCH_AXES: The logical axis of batch dimension. It is usually sharded along DP + FSDP on Mesh.
+* SEQLEN_AXES: The logical axis of sequence length dimension. It is usually not sharded.
+* SEQLEN_TP_AXES: The logical axis of sequence length dimension. It is usually not sharded along TP on Mesh.
```

Comment:
When using that, does it mean it is shareded over SEQLEN and TP? What TP mean here?

Reply:
Fixed typo. Basically, SEQLEN is not sharding, SEQLEN means sharding along TP mesh axis.